src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,221 @@
+
+### SpatialReference key allowed values
+
+| **Value name** | **Description**                                                                                                               |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------- |
+| orig           | A (potentially unique) per-image space. Useful for describing the source of transforms from an input image to a target space. |
+| uri or path    | This can be used to point to a specific file.                                                                                 |
+
+### Examples
+
+Preprocessed `bold` NIfTI file in `individual` coordinate space. Please mind
+that in this case `SpatialReference` key is REQUIRED.
+
+```Text
+sub-01/func/sub-01_task-rest_space-individual_bold.nii.gz
+sub-01/func/sub-01_task-rest_space-individual_bold.json
+```
+
+```JSON
+{
+    "SpatialReference": "/sub-01/anat/sub-01_desc-combined_T1w.nii.gz"
+}
+```
+
+Participant cortical thickness GIFTI file in `individual` coordinate space.
+Please mind that in this case `SpatialReference` key is REQUIRED.
+
+```Text
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.shape.gii
+sub-01/anat/sub-01_hemi-L_space-individual_thickness.json
+```
+
+```JSON
+{
+    "SpatialReference": "/sub-01/anat/sub-01_hemi-L_pial.surf.gii"
+}
+```
+
+## Metadata conventions
+
+-   Unless specified otherwise, individual sidecar JSON files and all metadata
+    fields within are optional. However, the appropriate use of these files and
+    pertinent fields is very valuable and thus encouraged. Moreover, for some
+    types of files, there may be one or more required metadata fields, in which
+    case at least one metadata file containing that field must be located
+    somewhere within the file’s hierarchy (per [the Inheritance
+    Principle](../02-common-principles.md#the-inheritance-principle)).
+
+-   When chaining derivative pipelines, any JSON fields that were specified as
+    mandatory in the input files should be propagated forward in the output
+    file’s JSON provided they remain valid. Non-required JSON fields can be
+    propagated, and are highly useful, but it is the pipeline’s responsibility
+    to ensure that the values are still relevant and appropriate to the type of
+    output data.
+
+## File naming conventions
+
+-   Filenames that are permissible for a raw BIDS data type have a privileged
+    status. Any modification of raw files must use a modified filename that does
+    not conflict with the raw filename. Further, any files created as part of a
+    derivative dataset must not match a permissible filename of a valid raw
+    dataset. Stated equivalently, if any filename in a derivative dataset has a
+    name permissible for a raw BIDS data, then that file must be an identical
+    copy of that raw file.
```

Comment:
Can anyone provide links to any prior conversations regarding this point? I suspect a lot of pipelines would provide, along with any derived image data, a sort of "minimally pre-processed" version of the input data, which could theoretically then be re-used for any of a wide range of things. While I see the appeal of forbidding name clashes between raw and minimally pre-processed data, it does beg the question: how would a pipeline creator then name such a file? Would they simply include some arbitrary keyword (e.g. "`_desc-`") in the filename, the only purpose of which would be to satisfy this criterion? Or could there be some reserved keyword for distinguishing between raw and minimally pre-processed data (since "`_desc-`" could plausibly appear in a raw BIDS data filename)?

The alternative would be that (in addition to the nature of the dataset itself) the contents of sidecar JSONs would distinguish between raw and derived data; e.g. if one contains acquisition parameters only, whereas the other contains fields related to pre-processing steps, it should be clear which is raw and which is derived, even if the filenames are identical.

Not wholly convinced of my own opinion here, so curious to know to what extent it's been contemplated.

Reply:
Additional relevant section later in `02-common-data-types.md` highlighted by @franklin-feingold below: 

> Note that even though `space` and `desc` are optional at least one of them needs to be defined to avoid name conflict with the raw file.

Reply:
Coming back to this now with more experience, I'd suggest that in the absence of any other identifying features, adding "`_desc-preproc`" to the pre-processed variant of image data is wholly appropriate and solves the issue; indeed this should even perhaps be a recommendation in this case.